articles/specs/launch.0.1.1.xsd

```diff
@@ -8,80 +8,82 @@
       </xs:documentation>
     </xs:annotation>
 
-    <xs:choice minOccurs="1" maxOccurs="unbounded">
-      <xs:annotation>
-        <xs:documentation xml:lang="en">
-          A collection of actions to be launched in order of appearance, plus
-          launch arguments for callers to provide, either through a tool or
-          by inclusion.
-        </xs:documentation>
-      </xs:annotation>
-
-      <xs:element name="arg">
+    <xs:complexType>
+      <xs:choice maxOccurs="unbounded">
         <xs:annotation>
           <xs:documentation xml:lang="en">
-            Declares launch file-level arguments.
+            A collection of actions to be launched in order of appearance, plus
+            launch arguments for callers to provide, either through a tool or
+            by inclusion.
           </xs:documentation>
         </xs:annotation>
 
-        <xs:complexType>
-          <xs:attribute name="name" type="xs:string" use="required">
-            <xs:annotation>
-              <xs:documentation xml:lang="en">
-                Name of the launch argument.
-              </xs:documentation>
-            </xs:annotation>
-          </xs:attribute>
-          <xs:attribute name="value" type="xs:string" use="optional">
-            <xs:annotation>
-              <xs:documentation xml:lang="en">
-                Fixed value for the launch argument, disables overrides.
-              </xs:documentation>
-            </xs:annotation>
-          </xs:attribute>
-          <xs:attribute name="default" type="xs:string" use="optional">
-            <xs:annotation>
-              <xs:documentation xml:lang="en">
-                Default value for the launch argument, used if non is provided.
-              </xs:documentation>
-            </xs:annotation>
-          </xs:attribute>
-          <xs:attribute name="description" type="xs:string" use="optional">
-            <xs:annotation>
-              <xs:documentation xml:lang="en">
-                Brief description of the launch argument.
-              </xs:documentation>
-            </xs:annotation>
-          </xs:attribute>
-        </xs:complexType>
-      </xs:element>
-      <xs:element ref="let"/>
-      <xs:element ref="node"/>
-      <xs:element ref="executable"/>
-      <xs:element ref="include"/>
-      <xs:element ref="group"/>
-      <xs:element ref="set_env"/>
-      <xs:element ref="unset_env"/>
-    </xs:choice>
-    <xs:attribute name="version" use="required">
-      <xs:annotation>
-        <xs:documentation xml:lang="en">
-          Launch XML schema version in use.
-        </xs:documentation>
-      </xs:annotation>
-      <xs:simpleType>
-        <xs:restriction base="xs:string">
-          <xs:pattern value="[0-9]+\.[0-9]+\.[0-9]+"/>
-        </xs:restriction>
-      </xs:simpleType>
-    </xs:attribute>
+        <xs:element name="arg">
+          <xs:annotation>
+            <xs:documentation xml:lang="en">
+              Declares launch file-level arguments.
```

Comment:
I guess this was already there, but could you explain a little more what is meant by `file-level` arguments?  It's not clear to me.

Reply:
> Declares a launch file argument.

conveys the same amount of information more clearly. Changed in  25db0215cb72dc84d9828dd25895593512f57582.